src/msw/renderer.cpp

```diff
@@ -788,6 +786,20 @@ wxRendererXP::DoDrawButtonLike(wxUxThemeHandle& hTheme,
     else if ( part == BP_PUSHBUTTON && (flags & wxCONTROL_ISDEFAULT) )
         state = PBS_DEFAULTED;
 
+    wxRect crect = rect;
+
+    // Checkboxes and radiobuttons must be drawn at native size, otherwise
```

Comment:
I'm not sure I agree with this comment, we don't do anything like this for any other elements, why should we do it here? IMO the caller should use the correct size, i.e. `GetCheckBoxSize()`, but if they want to use something else (e.g. `10*GetCheckBoxSize()` because they want a giant checkbox), we should respect their wishes.

Reply:
At 100% scale, if the rect is bigger than the checkbox, it's rendered size is still 13 px, it just get centered (or if we change scale at runtime from 100% to something else). This behaviour is replicated here.

It's probably a Windows bug that it gets scaled on non-100% display scale if we pass a bigger rectangle.

Reply:
>we don't do anything like this for any other elements, why should we do it here?

Push buttons are not affected by this issue. Checkboxes and radiobuttons are.

Reply:
Let's do this, but in wxPG itself and not here, like this it certainly won't break any other uses of this function.